guide/english/php/forms/handling-form-input/index.md

```diff
@@ -6,7 +6,8 @@ GET VS POST
 
 One can get form inputs with global variables $_POST and $_GET.
 ```
-$_POST["firstname"] or $_GET['lastname']
+$_POST['username'] or $_GET['username'] or $_REQUEST['username']
+
```

Comment:
Also there seems to be no introduction to request variable in the above lines, which makes it's introduction in the examples seem wayward.

Reply:
I have made changes. Let me know